src/app/(application)/experiences/(addExperiences)/actions.ts

```diff
@@ -1,38 +1,79 @@
-"use server"
-import { redirect } from 'next/navigation'
-
-
-type UrlValidationResult = {
-    isValid: true;
-  }
-| {
-    isValid: false;
-    url: string | undefined;
-    errorMessage: string;
-  }
+"use server";
+import { GetFareHarbourItemsForCompany } from "@/lib/api/fareharbour/fetcher/FareHarbourFetcher";
+import { FareHarbourCompany } from "@/lib/api/fareharbour/models/FareHarbourCompany";
+import { FareHarbourItemsResult } from "@/lib/api/fareharbour/models/FareHarbourItem";
+import { GetRezdySearchResultsFromMarketPlace } from "@/lib/api/rezdy/fetcher/RezdyFetcher";
 
+type UrlValidationResult =
+  | {
```

Comment:
Nitpick: | is not needed. Typo?

Reply:
`prettier` does this automatically when formatting the file